applications/metis_application/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h

```diff
@@ -239,11 +239,11 @@ class MetisDivideHeterogeneousInputInMemoryProcess : public MetisDivideHeterogen
             GraphColoringProcess(mNumberOfPartitions,DomainGraph,ColoredDomainGraph,NumColors).Execute();
 
             if (mVerbosity > 0) {
-                KRATOS_INFO(NumColors);
+                KRATOS_WATCH(NumColors);
```

Comment:
@sunethwarna I'd say the fix in #2617 is better here

Reply:
I agree :) I can revoke this pull request. You have everything handled in #2617. Only issue will be the changes I made in residual criteria.